test/e2e/network/service.go

```diff
@@ -852,11 +852,14 @@ var _ = common.SIGDescribe("Services", func() {
 					TargetPort: intstr.FromString(svc2port),
 				},
 			}
+			service.Spec.Type = v1.ServiceTypeNodePort
+			service.Spec.Selector = jig.Labels
+			service.ObjectMeta.Labels = jig.Labels
 		})
 		framework.ExpectNoError(err)
 
-		port1 := 100
-		port2 := 101
+		port1 := 8080
+		port2 := 8081
```

Comment:
no need to change this, the endpoint object have to do the connection , i.e
svcIP:svcPort ---> endpoint --> containerIP:containerPort